ci/linkcheck.sh

```diff
@@ -3,10 +3,17 @@
 set -e
 set -o pipefail
 
+LINKCHECK_BINARY=mdbook-linkcheck2
+
 set_github_token() {
   jq '.config.output.linkcheck."http-headers"."github\\.com" = ["Authorization: Bearer $GITHUB_TOKEN"]'
 }
 
+if [ ! -z "$SKIP_LINKCHECK" ] ; then
+  echo "Skipping link check."
+  exec $LINKCHECK_BINARY -f ""
```

Comment:
Shouldn't this be doing `exit 0` instead? I'm not sure why it's still running the linkcheck binary.

Reply:
Yep, adjusted.